resources/aws/launch_configuration.go

```diff
@@ -22,6 +23,13 @@ type LaunchConfiguration struct {
 	Client *autoscaling.AutoScaling
 }
 
+const (
+	defaultEBSVolumeMountPoint = "/dev/xvdh"
+	// defaultEBSVolumeSize is expressed in GB.
+	defaultEBSVolumeSize = 50
+	defaultEBSVolumeType = "gp2"
+)
```

Comment:
We discussed this with @rossf7 and he was saying he prefers having these as flags to the operator, which could be defined in architect.

I think this kind of value belongs to the TPO (since it's not really a property of the operator, but of the cluster).

I'll add an issue that mentions both approaches.